recipes/immutables/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set name = "immutables" %}
+{% set version = "0.5" %}
+{% set hash_type = "sha256" %}
+{% set hash_value = "57d17742fccec4c0466ab658053fab1a327df42241e7b1cab28e5b85e7066ac1" %}
+
+package:
+  name: '{{ name|lower }}'
+  version: '{{ version }}'
+
+source:
+  fn: '{{ name }}-{{ version }}.tar.gz'
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  '{{ hash_type }}': '{{ hash_value }}'
+
+build:
+  skip: True  # [py27]
+  number: 0
+  script: python -m pip install . --no-deps --ignore-installed
+
+requirements:
+  build:
+    - python
+    - pip
+    - setuptools
```

Comment:
This package looks like it uses c-extensions so please add `toolchain` as a `build` requirement.

Reply:
Looks like we reviewed at the same time and said the same things. :)

@nicoddemus, adding `toolchain` would also be fine, but https://github.com/conda-forge/staged-recipes/pull/5780#issuecomment-386094742 is the future-proof way that currently has the same effect.

Reply:
@dougalsutherland, Thanks for the link to the issue. Sometimes it's hard to keep up with best practices/know what's happening in all of the internal discussions on various repos.

Reply:
We know the feeling. :)

If you have any ideas about how we can communicate changes better, please let us know (perhaps as a new issue on the webpage repo).

Reply:
I did not know that, thanks for the pointers. Fixed.